react-backend/routes/cbauthentication/cb_register.js

```diff
@@ -7,7 +7,10 @@ router.post('/', (req, res, next) => {
   const { formPswd, formName, formEmail, formGenre } = req.body;
 
   const hashedPassword = hashCB(formPswd);
+  console.log('pw', hashedPassword);
```

Comment:
do we need these logs?

Reply:
Lol, that doesn't need to be a question.